src/controllers/room.controller.js

```diff
@@ -0,0 +1,30 @@
+const roomService = require('../services/room.service');
+
+const create = async (req, res) => {
+  const { name } = req.body;
+  const newRoom = await roomService.create(name);
```

Comment:
Consider adding error handling for the asynchronous operation. If the roomService.create method fails, it could throw an error that should be caught and handled appropriately.